README.md

```diff
@@ -1,4 +1,4 @@
-# Vite/Rollup plugin for Module Federation
+# Vite plugin for Module Federation
```

Comment:
For clarity will be good to mention that this library is Module Federation plugin for Vite based on [Module Federation 2](https://github.com/module-federation/core/discussions/2397) via `@module-federation/runtime` library.

Reply:
I updated the PR description. Feel free to open a PR for that. Thanks 💪

Reply:
Fixed